src/server/pps/cmds/cmds.go

```diff
@@ -235,6 +237,8 @@ $ pachctl list-job -p foo bar/YYY
 		}),
 	}
 	rawFlag(listDatum)
+	listDatum.Flags().BoolVar(&paginate, "paginate", false, "Get a subset of the results")
+	listDatum.Flags().Int64Var(&page, "page", 0, "Specify the page of results to send")
```

Comment:
I'm not sure we can do that ... bcz non initialized ints are 0 right? (And the pages are zero indexed) So I can't tell the difference between non initialized page and page=0 (the first page).

But ... this did make me realize it needs to be an unsigned int